packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -5,7 +5,11 @@ import {model, property} from '@loopback/repository';
 import {<%= modelBaseClass %>} from '.';
 <% } -%>
 
+<% if(disabledStrictMode) { -%>
```

Comment:
I think a more general way to get the model settings from the CLI is passing in a settings object, like

```ts
 <% if(settings) { -%>
 @model(<%= settings %>)
 <% } else { -%>
 @model()
 <% } -%>
```

just a suggestion :) can do it in a separate PR.